docker-compose.yml

```diff
@@ -1,13 +1,47 @@
 version: '3.7'
 
 services:
+  redis:
+    image: redis:6.2-alpine
+    restart: always
+    volumes:
+      - redis-data:/data
+
   cht-user-management:
-    image: public.ecr.aws/medic/cht-user-management:latest
+    build:
+      context: .
+      dockerfile: Dockerfile
     environment:
       - EXTERNAL_PORT=${EXTERNAL_PORT:-3000}
       - COOKIE_PRIVATE_KEY=${COOKIE_PRIVATE_KEY}
       - CONFIG_NAME=${CONFIG_NAME}
+      - REDIS_HOST=redis
+      - REDIS_PORT=6379
+      - MOVE_CONTACT_QUEUE=${MOVE_CONTACT_QUEUE:-MOVE_CONTACT_QUEUE}
```

Comment:
i don't think this `MOVE_CONTACT_QUEUE` value is actually loaded from env. it appears to be a hardcoded string. i agree it should not loaded from env